src/contributors/MapContributor.ts

```diff
@@ -303,18 +302,26 @@ export class MapContributor extends Contributor {
                             type: 'Feature',
                             geometry: geojsonWKT,
                             properties: {
-                                source: 'wkt',
-                                arlas_id: index
+                                source: 'wkt'
                             }
                         };
                         polygonGeojsons.push(feature);
                     }
                     index = index + 1;
```

Comment:
index is not needed anymore ?

Otherwise LGTM

Reply:
Indeed